crates/transaction-pool/src/pool/txpool.rs

```diff
@@ -3384,6 +3410,116 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_best_with_attributes() {
+        let on_chain_balance = U256::MAX;
+        let on_chain_nonce = 0;
+        let mut f = MockTransactionFactory::default();
+        let mut pool = TxPool::new(MockOrdering::default(), Default::default());
+
+        let base_fee: u64 = 100;
+        let blob_fee: u64 = 100;
```

Comment:
let's make them u128 so that we can avoid most casts

and then reintroduce them as u64